packages/flame_behaviors/example/pubspec.yaml

```diff
@@ -4,10 +4,10 @@ version: 1.0.0+1
 publish_to: none
 
 environment:
-  sdk: ">=2.17.0 <3.0.0"
+  sdk: ">=3.0.0 <4.0.0"
 
 dependencies:
-  flame: ^1.5.0
+  flame: ^1.8.0
```

Comment:
why not use 1.10 here too?

Reply:
Originally started as v1.8.0 upgrade, will change that!